packages/qwik-dom/lib/DOMImplementation.js

```diff
@@ -37,7 +37,7 @@ DOMImplementation.prototype = {
     //
     // Note that the current DOMCore spec makes it impossible to
     // create an HTML document with this function, even if the
-    // namespace and doctype are propertly set.  See this thread:
+    // namespace and doctype are property set.  See this thread:
```

Comment:
imo it should say `properly set` 😄 
```suggestion
    // namespace and doctype are properly set.  See this thread:
```